usaspending_api/common/helpers/generic_helper.py

```diff
@@ -113,12 +108,27 @@ def convert_matview_to_view(matview_sql):
     return sql
 
 
-def generate_matviews(materialized_views_as_traditional_views=False):
+def get_temp_matview_sql_files_dict(matview_dir: Path):
+    temp_sql_files = [
+        {"name": key, "sql_file": matview_dir / val["sql_filename"]} for key, val in MATERIALIZED_VIEWS.items()
+    ]
+    temp_sql_files += [
+        {"name": key, "sql_file": matview_dir / val["sql_filename"]} for key, val in CHUNKED_MATERIALIZED_VIEWS.items()
+    ]
+    return temp_sql_files
+
+
+def generate_matviews(materialized_views_as_traditional_views=False, parallel_worker_id: str = None):
```

Comment:
You should document the new parameter here, `parallel_worker_id`

Reply:
Done